intrastat_product/views/account_invoice.xml

```diff
@@ -18,7 +18,9 @@
                  attrs="{'invisible': [('intrastat', '!=', 'extended')]}"
                  widget="selection"/>
           <field name="src_dest_country_id" string="Destination Country"/>
-          <field name="src_dest_region_id" string="Origin Region" invisible="1"/>
+          <field name="src_dest_country_code" invisible="1"/>
+          <field name="src_dest_region_id" string="Origin/Destination Region"
+                 attrs="{'invisible': [('src_dest_country_code', 'not in', ['BE'])]}"/>
```

Comment:
I don't agree on that one.
I am a belgian company and I ship from Belgium to France. On the customer invoice:
src_dest_country_code = FR
src_dest_region_id = the Belgian region from where the goods are shipped
-> so it should not be invisible here !